cmd/server/main.go

```diff
@@ -21,13 +21,14 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/health/grpc_health_v1"
+	"google.golang.org/grpc/reflection"
 )
 
 var (
 	serverPort            = flag.Int("server_port", 3000, "grpc server port")
 	metricsPort           = flag.Int("metrics_port", 3001, "http metrics port")
 	logLevel              = flag.String("loglevel", utils.GetEnvVarString("LOGLEVEL", ""), "minimum log verbosity")
-	origin                = flag.String("origin", utils.GetEnvVarString("ORIGIN", ""), "ads.cert hostname for the originating party")
+	origin                = flag.String("origin", utils.GetEnvVarString("ORIGIN", ""), "ads.cert Call Sign domain name for this party's Signatory service deployment")
```

Comment:
@cmlight Are you keeping the `origin` name for the variable and flag?

Reply:
Yeah, at least for now.  I don't want to introduce a breaking change in case anyone has this deployed.  I'm wanting to experiment with a few configuration strategies for this server (especially given that it's getting other values passed in from the command line like private key which is something we'll want to move away from).